modes/reftex/evil-collection-reftex.el

```diff
@@ -121,6 +134,8 @@
 
   ;; This one is more involved, in reftex-toc.el, line 282 it shows the prompt
   ;; string with the keybinds and I don't see any way of changing it to show evil-like binds.
+  ;;;; Since bindings that are specified in the prompt are the same except for SPC, wouldn't
```

Comment:
Remove comment here.

Reply:
Got it, I will remove the comments once I have settled on the other keybindings to include